client/x11/10-ibus-x11.in

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+@libexecdir@/ibus-x11 &
```

Comment:
I think '--kill-daemon' option is needed since ibus-daemon does not exit when gnome-session & gnome-shell terminate.
Current ibus-x11 checks Display with XSetErrorHandler() and XSetIOErrorHandler().

Reply:
I think that escape hatch is generally not necessary anymore with systemd initialization :). All daemons should be tear down properly on session finalization.

In this case (starting the service together with Xwayland) it even sounds like it would lead to ibus-daemon unintendedly stopping in the middle of the session. ibus-x11 should shutdown individually when Xwayland is stopped.